bin.js

```diff
@@ -24,7 +22,7 @@ if (argv.help || argv.h) {
 	process.stdout.write(`
 Usage:
     vbb-anybar
-    vbb-anybar init <station-id> <next-station-id>
+    vbb-anybar init <station-id> <next-station-id> [<time-to-station>]
```

Comment:
Bonus points if you document `time-to-station` as an option, making clear that it is in minutes. 💛 

Refer to [another CLI tool from me](https://github.com/derhuerst/vbb-cli/tree/40813c593a4fb8383add8e54da0ddbe2413f72b5#usage) for an example.